src/python/module/smartredis/client.py

```diff
@@ -499,8 +499,8 @@ def run_script_multigpu(
         self,
         name: str,
         fn_name: str,
-        inputs: t.List[str],
-        outputs: t.List[str],
+        inputs: t.Optional[t.Union[str, t.List[str]]],
+        outputs: t.Optional[t.Union[str, t.List[str]]],
```

Comment:
This is the typehint needed in the earlier comments (possibly minus the t.Optional part)